prettier.config.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+module.exports = {
+  printWidth: 120,
+  tabWidth: 2,
+  useTabs: false,
+  semi: true,
+  singleQuote: true,
+  trailingComma: 'es5',
+  bracketSpacing: true,
```

Comment:
I'm being picky here, but if a config file can be JSON that seems more sensible to me.

Reply:
I will turn this into a json file.